org-formation/600-access/_tasks.yaml

```diff
@@ -413,6 +413,40 @@ SynapseLlmDeveloperPolicy:
             "Effect": "Allow",
             "Action": "iam:PassRole",
             "Resource": "*"
+          },
+          {
+            "Effect": "Allow",
+            "Action": [
+              "iam:CreateRole",
+              "iam:PutRolePermissionsBoundary"
+            ],
+            "Resource": "arn:aws:iam::${AWS::AccountId}:role/synllmrole-*",
+            "Condition": {
+              "StringEquals": {
+                "iam:PermissionsBoundary": "${synLlmLambdaExecPermissionBoundary}"
```

Comment:
ARN of boundary policy needs to be set here (had both resources in same template in original code).